gef.py

```diff
@@ -11603,6 +11587,7 @@ def reset_caches(self) -> None:
         "set print pretty on",
         "set disassembly-flavor intel",
         "handle SIGALRM print nopass",
+        "set print asm-demangle on",
```

Comment:
I do like this setting in 99% of the cases but in some cases it can slow GDB down incredibly. This should be a default IMO but we should be able to modify it through gef settings...

Reply:
I dont have any benchmarks or anything but I remember working on some binaries with a whole lot of necessary demangling where I had to wait about 1 minute or something before I could interact with gdb because of that setting (at least it stopped when I changed that setting)

Reply:
Removed for now, I realized it's out of scope anyway.
We can look into that later